_backend/config.example.php

```diff
@@ -4,17 +4,17 @@
 
 return array(
   'release_title'    => 'Odin',
-  'release_version'  => '6',
-  'release_filename' => 'elementaryos-6.0-stable.20210901.iso',
-  'release_size'     => '2.49 GB',
-  'release_magnet'   => '73e9c0288c0b62c2646b695219b550fd231fede4',
-  'release_sha256'   => '439110ceadadd13683632ddd273095f72ee9b8d22a7569ac84c2fd1c4d920610',
+  'release_version'  => '6.0',
```

Comment:
If it’s not required, I would like to keep this clean at `6`, since that’s how we refer to this release everywhere.